File: java/notas.java
import java.util.Locale;
import java.util.Scanner;

public class notas {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double nota1, nota2, notafinal;

	    System.out.print("Digite a primeira nota: ");
	    nota1 = sc.nextDouble();

	    System.out.print("Digite a segunda nota: ");
	    nota2 = sc.nextDouble();

	    notafinal = nota1 + nota2;

	    System.out.printf("NOTA FINAL = %.1f\n", notafinal);

	    if (notafinal < 60.0) {
	    	System.out.println("REPROVADO");
	    }

		sc.close();
	}
}

File: python/notas.py
nota1: float; nota2: float; notafinal: float

nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))

notafinal = nota1 + nota2

print(f"NOTA FINAL = {notafinal:.1f}")

if notafinal < 60.0:
	print("REPROVADO")